tc_driver/Cases.jl

```diff
@@ -575,15 +517,14 @@ function initialize_profiles(
     @. aux_gm.tke = prof_tke(z)
 end
 
-function surface_params(case::TRMM_LBA, surf_ref_thermo_state, thermo_params)
-    p_f_surf = TD.air_pressure(thermo_params, surf_ref_thermo_state)
-    FT = eltype(p_f_surf)
+function surface_params(case::TRMM_LBA, thermo_params)
+    FT = eltype(thermo_params)
     # zrough = 1.0e-4 # not actually used, but initialized to reasonable value
     zrough::FT = 0 # actually, used, TODO: should we be using the value above?
+    psurface::FT = 991.3 * 100
     qsurface::FT = 22.45e-3 # kg/kg
-    θ_surface::FT = (273.15 + 23)
-    ts = TD.PhaseEquil_pθq(thermo_params, p_f_surf, θ_surface, qsurface)
-    Tsurface = TD.air_temperature(thermo_params, ts)
+    θ_surface::FT = (273.15 + 23) # PAPER CHECK: Should we use Tsurface = 296.85 instead?
```

Comment:
T_surface = 296.85 is the correct one. In the paper it is 23.7 C. So looks like we have lost 0.7 C somehow